tests/test_batch.py

```diff
@@ -20,13 +20,13 @@
     JobRetryError,
     RebatchError,
 )
-from tests.test_doubles.authentication import FakeAuth0AuthenticationSuccess
 from tests.conftest import mock_core_response
+from tests.test_doubles.authentication import FakeAuth0AuthenticationSuccess
 
 
 class TestBatch:
     @pytest.fixture(autouse=True)
-    def mock_sleep(self):
+    def _mock_sleep(self):
```

Comment:
`ruff rule PT004`:

> By convention, fixtures that don't return a value should be named with a
leading underscore, while fixtures that do return a value should not.

Didn't know